src/frontend/src/App.js

```diff
@@ -262,6 +266,47 @@ function App() {
     });
   };
 
+  const handleAddAnnotationMultiSelect = (hexagonIDs) => {
+
+    setAnnotationHexagonIDs((prevAnnotationHexagonIDs) => {
+      const newAnnotationHexagonIDs = {
+        presence: new Set(prevAnnotationHexagonIDs.presence),
+        absence: new Set(prevAnnotationHexagonIDs.absence),
+      };
+
+      // Calculate the current hexagons for the specified annotation type
+      const curHexIDs = newAnnotationHexagonIDs[annotationType];
+      // Determine the threshold count for deciding between adding or removing hexagons
+      const hexThresholdCount = parseInt(hexagonIDs.length * 0.35);
```

Comment:
i am curious about the logic going on here, hexthreshold is used to determine addition or removing, is the logic if it covers -- ok yall explained to me

Reply:
removed this part in the new commit